package.json

```diff
@@ -20,6 +20,7 @@
     "cross-env": "^5.0.5",
     "electron": "~1.7.8",
     "electron-builder": "^19.41.1",
+    "electron-prebuilt": "^1.4.13",
```

Comment:
It is really required `electron-prebuilt` in order to run Spectron in windows? I would like to avoid extra dependencies if possible

Reply:
Unfortunately I could not find another way to resolve the issue with Spectron on Windows. What I have discovered is that './node_modules/.bin/electron' should work properly but has a bug somewhere downstream.

I will do some more research and see if there is a way around this issue without adding dependencies.